src/helpers/commandIdCache.ts

```diff
@@ -0,0 +1,10 @@
+import {Collection} from "discord.js";
+
+/** Maps a Guild onto a map from name to ID */
+export const GuildCommandIDs = new Collection<
```

Comment:
I think explicit 
```suggestion
const GuildCommandIDs = new Collection<
...
export { GuildCommandIDs};
```
is better because at a quick glance, you can see everything that is exported from a file and change it nicely in a single spot. It makes it much easier for managing files and exports

Reply:
BTW this goes for all exports in your PR

Reply:
I made this change